app/models/notification.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: notifications
+#
+#  id             :bigint           not null, primary key
+#  params         :jsonb
+#  read_at        :datetime
+#  recipient_type :string           not null
+#  type           :string           not null
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  recipient_id   :bigint           not null
+#
+# Indexes
+#
+#  index_notifications_on_read_at    (read_at)
+#  index_notifications_on_recipient  (recipient_type,recipient_id)
+#
+class Notification < ApplicationRecord
+  include Noticed::Model
+  belongs_to :recipient, polymorphic: true
```

Comment:
Why is this a polymorphic relation?

Reply:
This is a clean way to have notifications for severals models.
Whit this relation, It won't be required to create a relation for each model.
In this case we only have 1 model, but after I searched severals ways to implement this gem, I find this as a good and readable one. @JulianPasquale

Reply:
Ok so you set this as a polymorphic relation just in case you want to use this in the future for another model?

I cannot think about another use case for Notifications in this target app. Can you please provide an example where you would use this in the future? Who can be notified besides users?